src/transformation/to_trimesh/cuboid_to_trimesh.rs

```diff
@@ -2,11 +2,20 @@ use crate::bounding_volume::Aabb;
 use crate::math::Real;
 use crate::shape::Cuboid;
 use crate::transformation::utils;
+
+#[cfg(feature = "dim3")]
+use na::Point3 as PointDim;
+#[cfg(feature = "dim3")]
 use na::{self, Point3};
 
+#[cfg(feature = "dim2")]
+use na::Point2 as PointDim;
+#[cfg(feature = "dim2")]
+use na::{self, Point2};
+
```

Comment:
You can remove these, and just use `crate::math::Point` instead of `PointDim/Point2/Point3`.